From 3eeec4e1a646f60b136a431338984ae78e85ff00 Mon Sep 17 00:00:00 2001
From: Jeff Kudrick <jeff.kudrick@freescale.com>
Date: Mon, 12 Mar 2012 15:48:15 -0500
Subject: [PATCH 253/498] ENGR00175612: Camera preview window is overlapped
 with the picture

Removed debug print from iMX5 Camera HAL libraries.

Signed-off-by: Jeff Kudrick <jeff.kudrick@freescale.com>
---
 mx5x/libcamera/Camera_utils.h | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/mx5x/libcamera/Camera_utils.h b/mx5x/libcamera/Camera_utils.h
index dfca274..1ce8a7c 100755
--- a/mx5x/libcamera/Camera_utils.h
+++ b/mx5x/libcamera/Camera_utils.h
@@ -27,20 +27,22 @@
 #include <utils/Log.h>
 #include <utils/threads.h>
 
-#define CAMERA_HAL_DEBUG_LOG
+//#define CAMERA_HAL_DEBUG_LOG
 
 #ifdef CAMERA_HAL_DEBUG_LOG
 #define CAMERA_HAL_LOG_RUNTIME(format, ...) LOGI((format), ## __VA_ARGS__)
 #define CAMERA_HAL_LOG_FUNC LOGI("%s is excuting...",  __FUNCTION__)
+#define CAMERA_HAL_LOG_TRACE   LOGI("%s : %d", __FUNCTION__,__LINE__)
+#define CAMERA_HAL_LOG_INFO(format, ...) LOGI((format), ## __VA_ARGS__)
 #else
 #define CAMERA_HAL_LOG_RUNTIME(format, ...) 
 #define CAMERA_HAL_LOG_FUNC
+#define CAMERA_HAL_LOG_TRACE
+#define CAMERA_HAL_LOG_INFO(format, ...)
 #endif
 
-#define CAMERA_HAL_LOG_TRACE   LOGI("%s : %d", __FUNCTION__,__LINE__)
-#define CAMERA_HAL_LOG_INFO(format, ...) LOGI((format), ## __VA_ARGS__)
-
 #define CAMERA_HAL_ERR(format, ...) LOGE((format), ##__VA_ARGS__)
+
 namespace android {
 
     typedef enum{
-- 
1.8.0

